src/__tests__/usePromiseState.ts

```diff
@@ -0,0 +1,125 @@
+import { renderHook } from '@testing-library/react-hooks';
```

Comment:
Since `usePromise` doesn't dispatch any render update, what do you think about renaming this test file to just `usePromise` without the `State` suffix?